src/gt4py/ir/nodes.py

```diff
@@ -684,8 +689,11 @@ def full_interval(cls, order=IterationOrder.PARALLEL):
 
 @attribclass
 class ComputationBlock(Node):
+    # TODO Should this be renamed to "sequential_interval"?
     interval = attribute(of=AxisInterval)
+    # TODO Should this be renamed to "sequential_interation_order"?
```

Comment:
Once these are answered, you should consider removing the comments and possibly opening an issue